bolt_mujoco_simulation/etc/robot.xml

```diff
@@ -2,6 +2,9 @@
   <body name="robot" pos="0 0 0">
     <geom name="body_visual" type="mesh" contype="0" conaffinity="0" group="1" density="0" rgba="0.8 0.8 0.8 1" mesh="bolt_body"/>
     <geom name="body_collision" type="mesh" rgba="0.8 0.8 0.8 1" mesh="bolt_body"/>
+    <body name="sensor_body" pos="0 0 0">
```

Comment:
i would prefer something like `sensor_link`. also, this should also be added to the urdf model

Reply:
done